libmachine/cert/cert.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/rancher/machine/libmachine/auth"
 	"github.com/rancher/machine/libmachine/log"
-	u "github.com/rancher/machine/libmachine/util"
+	util "github.com/rancher/machine/libmachine/util"
```

Comment:
nit: if we are naming this `util`, then this can be removed.
```suggestion
	"github.com/rancher/machine/libmachine/util"
```